### PR TITLE
Printers run dry

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
The endless magic money of the department payouts have been disabled.

### CODE REVIEW

**Comment by r4d6** [comment]
Look at https://github.com/sojourn-13/sojourn-station/pull/2334
You don't need to change a chunk of the code, only the department's `funding_type` to `FUNDING_NONE`
But I do disagree with doing that.

File: code/controllers/subsystems/economy.dm [hunk 1]
```diff
@@ -41,20 +41,21 @@ SUBSYSTEM_DEF(economy)
...
+			if(!ED) // If no employer department found - payment is external
+				deposit_to_account(A, A.employer, "Payroll Funding", "Hansa payroll system", amount_to_pay)
+				paid_external += amount_to_pay
+				continue
 			else
-				A.debt += A.wage
-				ED.total_debt += A.wage
+				var/datum/money_account/EA = get_account(ED.account_number)
+				if(amount_to_pay <= EA.money)
+					transfer_funds(EA, A, "Payroll Funding", "CEV Eris payroll system", amount_to_pay)
```
Comment (ThePainkiller) [inline code/controllers/subsystems/economy.dm:52]: "CEV Eris payroll system" could use a refactor....

Comment (Trilbyspaceclone) [reply code/controllers/subsystems/economy.dm:52]: Its being like this for debugging reasons.

**Comment by Trilbyspaceclone** [comment]
image

**Pull request merged**